content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -37,7 +37,16 @@ by providing the following flags to the kube-apiserver:
 1. Create an egress configuration file such as `admin/konnectivity/egress-selector-configuration.yaml`.
 1. Set the `--egress-selector-config-file` flag of the API Server to the path of
 your API Server egress configuration file.
-
+1. Add voluimens config to the kube-apiserver:
```

Comment:
volumes rather than voluimens?
Also this step is specifically (only) needed for UDS connections. We should indicate that in the note.

Reply:
Now the whole demo uses UDS.

Reply:
Agreed as its easier to maintain a single path for demonstration. However it is not the only supported option. Their is for instance an option to use mTLS & GRPC. That having been said you seem to have added the indicator, thank you :D

Reply:
/lgtm

Reply:
Done.